eZ/Publish/SPI/FieldType/ValidationError.php

```diff
@@ -22,5 +22,25 @@
  */
 interface ValidationError extends Translatable
 {
+    /**
+     * Sets the target element on which the error occurred.
+     *
+     * E.g. Property of a Field value which didn't validate against validation.
+     * Can be a property path compatible with Symfony PropertyAccess component.
+     *
+     * Examples:
+     * - "[StringLengthValidator][minStringLength]" => Target is "minStringLength" key under "StringLengthValidator" key (fieldtype validator configuration)
+     * - "my_field_definition_identifier"
+     *
+     * @param string $target
+     */
+    public function setTarget($target);
```

Comment:
i'm a bit lost these days with coding standards... shouldn't we have still spaces before/after parenthesis in this repo? or all the new code can be psr2?